src/path-symex/path_symex.cpp

```diff
@@ -123,6 +125,11 @@ void path_symext::assign(
     {
       // done in statet:instantiate_rec
     }
+    else if(statement == ID_gcc_builtin_va_arg_next){
```

Comment:
White-space styling: no space around ==, put { on a new line, drop the blank line